test/unit/operations/indexes.test.ts

```diff
@@ -0,0 +1,143 @@
+import { expect } from 'chai';
+
+import {
+  CreateIndexesOptions,
+  CreateIndexOperation,
+  IndexDirection
+} from '../../../src/operations/indexes';
+import { ns } from '../../../src/utils';
+
+describe('makeIndexSpec()', () => {
+  const testCases = [
+    {
+      description: 'single string',
```

Comment:
Can we add tests for the index name as well?

Reply:
Is that not what the `should set name correctly...` it block does?

Reply:
I think Bailey is looking for a test case where the name *is* explicitly provided

Reply:
added!